apps/company/serializers.py

```
@@ -248,34 +308,566 @@ class Meta:
+                    results[name][date.strftime("%Y-%m-%d")] = 0
+        
         return results
 
-    def get_product_count(self, obj):
-        return Product.objects.filter(stocks__company=obj).distinct().count()
+class ValidateDict(serializers.Serializer):
+    sorting_warehouse_id = serializers.IntegerField()
+    stock = serializers.IntegerField()
+
+    def validate(self, attrs):
+        if not ("sorting_warehouse_id" in attrs.keys() and "stock" in attrs.keys()):
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)